libafl/src/corpus/testcase.rs

```diff
@@ -497,7 +499,13 @@ impl<I> Drop for Testcase<I> {
     fn drop(&mut self) {
         if let Some(filename) = &self.filename {
             let mut path = PathBuf::from(filename);
-            let lockname = format!(".{}.lafl_lock", path.file_name().unwrap().to_str().unwrap());
+            let lockname = if let Some(name) = path.file_name().and_then(|os_str| os_str.to_str()) {
+                format!(".{name}.lafl_lock")
+            } else {
+                // Log an error or silently return if conversion fails.
+                eprintln!("Failed to convert filename to string for lock file.");
+                return;
```

Comment:
I think this should panic, it's a bug in the filename / fuzzer. Maybe use `expect` then?

Reply:
(It's also _highly_ unlikely to ever happen, but still..)